File: archived/src/go/ch01/1.2/l0/demo_basic_ops.go
package main

import "fmt"

func main() {
	// Addition
	a, b := 478, 259
	fmt.Printf("Addition: %d + %d = %d\n", a, b, a+b)

	// Subtraction
	a, b = 503, 78
	fmt.Printf("Subtraction: %d - %d = %d\n", a, b, a-b)

	// Multiplication
	a, b = 214, 3
	fmt.Printf("Multiplication: %d * %d = %d\n", a, b, a*b)

	// Integer division + remainder
	n, d := 47, 5
	q, r := n/d, n%d
	fmt.Printf("Division: %d / %d -> quotient %d, remainder %d\n", n, d, q, r)
}

File: archived/src/c/ch01/1.2/l0/demo_basic_ops.c
#include <stdio.h>

int main(void) {
    int a = 478, b = 259;
    printf("Addition: %d + %d = %d\n", a, b, a + b);

    a = 503; b = 78;
    printf("Subtraction: %d - %d = %d\n", a, b, a - b);

    a = 214; b = 3;
    printf("Multiplication: %d * %d = %d\n", a, b, a * b);

    int n = 47, d = 5;
    int q = n / d;
    int r = n % d;
    printf("Division: %d / %d -> quotient %d, remainder %d\n", n, d, q, r);

    return 0;
}
